app/views/inventories/show.html.erb

```diff
@@ -0,0 +1,37 @@
+<p style="color: green"><%= notice %></p>
+
+<section class = 'd-flex flex-column mb-2 align-items-center'>
+<div class="section-title inventory_foods">
+  <h1>
+    <%= @inventory.name %>
+  </h1>
+    <%= link_to icon('fa-solid', 'plus'), "/inventory_foods/#{@inventory.id}/new", method: :post, class:'btn btn-add'%>
+  </div>
+
+  <div class='table'id="table-container">
+    <table class = 'mb-2 align-items-center'>
+    <tbody>
+      <tr>
+        <th>Food</th>
+        <th>Quantity</th>
+        <th>Actions</th>
+      </tr>
+        <% if @inventory_food.length > 0 %>
+          <% @inventory_food.each do |f| %>
+            <tr>
+              <td><%= f.food.name %></td>
+              <td><%= f.quantity %> <%= f.food.measurement_unit %></td>
+              <td><%= link_to icon('fa-solid', 'trash-can'), inventory_inventory_food_path(@inventory.id,f.id), data: { "turbo-method": :delete }, class: "btn btn-outline-danger" %></td>
```

Comment:
[REQUIRED] I can't see any links here. Kindly ensure that when an inventory item is clicked it leads to the form that allows a user to add food. This is a requirement for this milestone.